sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/files/oems/ec2

```diff
@@ -0,0 +1,36 @@
+/oem/base/README
+/oem/base/base.ign
+/oem/bin/amazon-ssm-agent
+/oem/bin/install-nvidia
+/oem/bin/setup-nvidia
+/oem/bin/ssm-cli
+/oem/bin/ssm-document-worker
+/oem/bin/ssm-session-logger
+/oem/bin/ssm-session-worker
+/oem/eks/bootstrap.sh
+/oem/eks/docker-daemon.json
+/oem/eks/download-kubelet.sh
+/oem/eks/eni-max-pods.txt
+/oem/eks/kubelet-config.json
+/oem/eks/kubelet-kubeconfig
+/oem/eks/kubelet.service
+/oem/ssm/amazon-ssm-agent.json.template
+/oem/ssm/seelog.xml.template
+/oem/units/nvidia.service
+/etc/amazon/ssm/amazon-ssm-agent.json
+/etc/amazon/ssm/seelog.xml
+/etc/eks/bootstrap.sh
+/etc/systemd/system/amazon-ssm-agent.service
+/etc/systemd/system/nvidia.service
+/etc/systemd/system/multi-user.target.wants/amazon-ssm-agent.service
+/etc/systemd/system/multi-user.target.wants/nvidia.service
+/etc/systemd/system/multi-user.target.requires/coreos-metadata-sshkeys@core.service
+/oem/grub.cfg
+/oem/oem-release
+/oem/base
+/oem/bin
+/oem/eks
+/oem/ssm
+/oem/units
+/etc/amazon
```

Comment:
```suggestion
```
see above, the files might be modified and while we support updating them for new deployments through the symlink mechanism we don't have a way to identify if we can clean this up, so it's only opt-in. Maybe good for the release notes to state that deleting the files once is needed for auto updating.